### PR TITLE
Fix TransactionResource with initial status and type

### PR DESCRIPTION
**Pull request by vmaletta**
Please merge

### CODE REVIEW

File: src/main/java/org/ibp/api/java/impl/middleware/inventory/manager/TransactionServiceImpl.java [hunk 1]
```diff
@@ -37,4 +39,14 @@ public Integer saveTransaction(final TransactionDto transactionDto) {
 		this.transactionInputValidator.validate(transactionDto);
 		return this.transactionService.saveTransaction(transactionDto);
 	}
+
+	@Override
+	public List<TransactionStatus>  getAllTransactionStatuses() {
```
Comment (clarysabel) [inline src/main/java/org/ibp/api/java/impl/middleware/inventory/manager/TransactionServiceImpl.java]: Even when statuses is a valid plural, it is uncommon. I think we should use status for plural

File: src/main/java/org/ibp/api/rest/inventory/manager/TransactionResource.java [hunk 2]
```diff
@@ -122,6 +124,9 @@ public long getCount() {
 		if (transactionDto.getLot() == null) {
 			transactionDto.setLot(new ExtendedLotDto());
 		}
+		//FIXME when this resource is completed
+		transactionDto.setTransactionStatus(TransactionStatus.CONFIRMED.getValue());
```
Comment (clarysabel) [inline src/main/java/org/ibp/api/rest/inventory/manager/TransactionResource.java]: Mention https://ibplatform.atlassian.net/browse/IBP-3455

File: src/main/java/org/ibp/api/java/impl/middleware/inventory/manager/TransactionServiceImpl.java (hunk 1 above)
Comment (vmaletta) [reply src/main/java/org/ibp/api/java/impl/middleware/inventory/manager/TransactionServiceImpl.java]: Done

File: src/main/java/org/ibp/api/rest/inventory/manager/TransactionResource.java (hunk 2 above)
Comment (vmaletta) [reply src/main/java/org/ibp/api/rest/inventory/manager/TransactionResource.java]: Done

**Pull request merged**